src/composables/useSignRelayerApproval.ts

```diff
@@ -49,6 +53,15 @@ export default function useSignRelayerApproval(relayerType: RelayerType) {
     isSignAction: true,
   };
 
+  function getAction() {
+    const provider = lsGet('connectedProvider');
```

Comment:
Instead of getting this from local storage, can we get it from the active state in useWeb3?